package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "*",
```

Comment:
Why do you have a `*` here? 
You should define a version range that works with the rest of material-ui libraries